### ISSUE TITLE
Make mac addr checks disabled by default and add parameter to turn on

### ISSUE THREAD

**Issue by jadilet** [subutai-io/vagrant#141]
@akarasulu commented on [Sat Jan 13 2018](https://github.com/subutai-io/packer/issues/30)

Right now the default is to check for mac addresses on the network after creating a random mac address to be absolutely certain that no other machine has the same mac address on the network.

This is not always needed. I do not think this should be enabled for the simple default case. There should be a parameter to turn this feature on like MAC_CHECK which can be either true or false. The default should be false.

**Comment by akarasulu** [comment]
@jadilet I agree with the MAC_CHECK but perhaps name it something more clear like ENABLE_MAC_CHECK ... boolean parameter is clear.

**Comment by jadilet** [comment]
Got it, I will change the configuration parameter to `ENABLE_MAC_CHECK`.

**Comment by Almansherov** [comment]
Implemented in master:

![image](https://user-images.githubusercontent.com/32430693/52325065-45df8880-29a9-11e9-8665-9b879ce1b430.png)
